### ISSUE TITLE
Misspelling Audit constructor parameter

### ISSUE THREAD

**Issue by zivkaziv** [lirantal/is-website-vulnerable#36]
```
module.exports = class Audit {
  constructor(lighthoutSettings) {
    this.settings = lighthoutSettings || {
      onlyAudits: ['no-vulnerable-libraries', 'js-libraries']
    }
  }
```
Misspelled parameter. Should be lighthouseSettings and not lighthoutSettings

## Expected Behavior
n/a

## Current Behavior
n/a

## Possible Solution
Replace lighthoutSettings to lighthouseSettings

## Steps to Reproduce (for bugs)
n/a

## Context
n/a

## Your Environment
n/a

- Library Version used:
- Node.js version (e.g. Node.js 5.4):
- Operating System and version (desktop or mobile):

**Comment by lirantal** [comment]
Good catch

**Comment by lirantal** [comment]
@zivkaziv can you open a PR to fix this?

**Comment by zivkaziv** [comment]
Sure.. You can sign on me

**Comment by lirantal** [comment]
Fixed by #40
